lib/rules/comment-pattern/index.js

```diff
@@ -0,0 +1,47 @@
+// @ts-nocheck
+
+'use strict';
+
+const _ = require('lodash');
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+
+const ruleName = 'comment-pattern';
+
+const messages = ruleMessages(ruleName, {
+	expected: 'Expected comment to match specified pattern',
```

Comment:
I think message would be more understandable if we show pattern in it:

```
Expected comment to match specified pattern: /foo.+/
```

What do you think, @stylelint/contributors?